Justfile

```
@@ -111,7 +111,16 @@ do-it:
     echo "You've done it!"
 
 # Clean up the environment by removing the artifacts and cache folders and running the forge clean command
+# Usage: just clean
 clean:
     echo "Cleaning up environment..."
     rm -rf artifacts cache
-    forge clean
\ No newline at end of file
+    forge clean
+
+# Fully clean the environment by removing the artifacts, the dependencies, and cache folders and running the forge clean-all command
+# Usage: just clean-all
+clean-all:
+    echo "Cleaning up environment..."
+    rm -rf artifacts cache
+    forge clean
+    rm -rf node_modules
```

```suggestion
    rm -rf node_modules
    forge remove vibc-core-smart-contracts
```
Wonder if this would make more sense? To have a full sweep?

---

Let's leave it as is for now